### PR TITLE
new packages: python-selenium-is-seledroid, python-seledroid, seledro…

### PR DESCRIPTION
**Pull request by nacho00112**
`python-selenium-is-seledroid:` selenium dummy that imports seledroid.
`python-seledroid:` Automatically install the seledroid library and the app.
`seledroid-app:` Installs the seledroid app.

I taked the idea from the issue i opened here https://github.com/termux-user-repository/tur/issues/375

### CODE REVIEW

**Comment by T-Dynamos** [comment]
@nacho00112 You can squash commits by this method:
Open a terminal and navigate to the Git repository that contains the commits you want to squash.

1. Run the following command to start an interactive rebase:

```console
git rebase -i HEAD~<number of commits to squash>
```
2. Replace <number of commits to squash> with the number of commits you want to squash. For example, if you want to squash the last 3 commits, use HEAD~3.

3. Git will open an editor with a list of the selected commits. The list will look something like this:

```console
pick e64f4d4 Commit message 1
pick 2a2b5f9 Commit message 2
pick 6c7d8e1 Commit message 3
```
4. Change the word "pick" to "squash" or "s" for the commits you want to squash. For example, if you want to squash the last 3 commits, the list should look like this:

```console
pick e64f4d4 Commit message 1
squash 2a2b5f9 Commit message 2
squash 6c7d8e1 Commit message 3
```

5. Save and close the editor.

6. Git will prompt you to enter a new commit message for the squashed commit. Edit the message if necessary, save and close the editor.

7. Git will automatically update the branch with the squashed commit.

8. If you have already pushed the original commits to a remote repository, you will need to force push the changes with the following command:

```console
git push --force
```

**Comment by nacho00112** [comment]
I thought i was getting error because i did not add "--force", i did `git pull` as it said me, but that command puts all the commits again plus the new commit with the squashes, thanks you, the checks seems to work except the i686, is that well?

**Comment by T-Dynamos** [comment]
@nacho00112 i686 crash will be fixed soon, then PR will be merged.

**Comment by nacho00112** [comment]
Fixed some bug i didn't fixed last night, now i built it and it's working

**Comment by nacho00112** [comment]
Seems all checks build now, i finished of adding commits, i tried to add an `uninstall-seledroid-app` script but i can't find information in google

**Comment by nacho00112** [comment]
I am working in another package `app-utils` without root with a tool to list the apps installed by their name or by their package domain

also another tool that check if an application is installed, by taking the name or package domain as argument, and an option to check his version

another tool to get the apk of an installed application by also taking the or package domain

a tool to show the app name or package domain of an apk

and finally another tool that install apks by temporarily disabling `allow-external-apps` in `~/.termux/termux.properties`

**Comment by nacho00112** [comment]
I mainly use python so my bash isn't very good but i will try to do it in bash so python not have to be installed

File: tur/python-seledroid/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/luanon404/seledroid
+TERMUX_PKG_DESCRIPTION="Python library to control android webdriver."
+TERMUX_PKG_LICENSE="Unlicense"
+TERMUX_PKG_MAINTAINER="@nacho00112"
+TERMUX_PKG_VERSION=1.1.0
+TERMUX_PKG_SRCURL=https://github.com/luanon404/seledroid/archive/refs/heads/main.zip
```
Comment (licy183) [inline tur/python-seledroid/build.sh]: Do not use `main.zip`. This is not a proper way when packaging a package. If any change is pushed to the main branch and then the checksum will change too.

File: tur/seledroid-app/build.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/luanon404/android-webdrivers
+TERMUX_PKG_DESCRIPTION="Chromium-based android webdriver application used by seledroid."
+TERMUX_PKG_LICENSE="Unlicense"
+TERMUX_PKG_MAINTAINER="@nacho00112"
+TERMUX_PKG_VERSION=1.1.1
+TERMUX_PKG_SRCURL=https://github.com/luanon404/android-webdrivers/archive/refs/heads/main.zip
```
Comment (licy183) [inline tur/seledroid-app/build.sh]: Same here.

File: tur/seledroid-app/build.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	chmod +x $INSTALL_SELEDROID_APP_FILE
+	cp $INSTALL_SELEDROID_APP_FILE $TERMUX_PREFIX/bin
+}
+
+termux_step_create_debscripts() {
+	echo '
+echo "INFO: You can still install the SeleDroid APP later by executing \`install-seledroid-app\`"
+echo "INFO: Also you can get the APK from '"$SELEDROID_APK"'"
+
+install-seledroid-app
```
Comment (licy183) [inline tur/seledroid-app/build.sh]: Run this script in `postinst` is not a good choice. In the script you're requesting the response of user, so sometimes `postinst` will get stuck in and then the package environment will be broken.

**Review by licy183 — CHANGES_REQUESTED** [review]

File: tur/python-seledroid/build.sh (hunk 1 above)
Comment (nacho00112) [reply tur/python-seledroid/build.sh]: I forked the repo to solve the problem

File: tur/seledroid-app/build.sh (hunk 2 above)
Comment (nacho00112) [reply tur/seledroid-app/build.sh]: Changed it to a fork

File: tur/seledroid-app/build.sh (hunk 3 above)
Comment (nacho00112) [reply tur/seledroid-app/build.sh]: I removed it, now the user have to execute it manually

**Pull request merged**